locations/spiders/remax_fr.py

```diff
@@ -0,0 +1,66 @@
+import json
+import re
+
+import scrapy
+
+from locations.items import GeojsonPointItem
+
+
+class RemaxFrSpider(scrapy.Spider):
+    name = "remax_fr"
+    item_attributes = {
+        "brand": "RE/MAX",
+        "brand_wikidata": "Q965845",
+    }
+    allowed_domains = ["remax.fr", "s.maxwork.fr"]
+
+    def start_requests(self):
+        url = "https://www.remax.fr/Api/Office/Search?size=2000"
+        payload = json.dumps(
+            {
+                "filters": [
+                    {"field": "OfficeName", "type": 0, "fuzziness": None},
+                    {"field": "Region2ID", "type": 0},
+                    {"field": "LanguagesSpokenIds", "type": 0},
+                ]
+            }
+        )
+        headers = {
+            "content-type": "application/json",
+            "languageid": "2",
+            "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36",
+        }
+        yield scrapy.Request(url=url, headers=headers, method="POST", body=payload, callback=self.parse)
+
+    def parse(self, response):
+        url = "https://s.maxwork.fr/site/static/2/offices/details_V2/{}.html"
+        for data in response.json().get("results"):
+            yield scrapy.Request(
```

Comment:
Note for future: you can use `JsonRequst` and it'll do the headers for you, and do the dict -> string too